src/store/store.ts

```diff
@@ -588,17 +588,20 @@ export const useStore = defineStore('store', {
 
     /** Whether all the amalgamation (regular) steps are valid. */
     isAmalgamationRegularValid (): boolean {
-      // *** TODO: add checks for review page components
-      const isFolioNumberValid = !this.isPremiumAccount || this.getFolioNumberValid
+      const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
```

Comment:
Found this missing. Now we're not able to file and pay if the review and confirm page is invalid.